frontend/app/components/shared/Filters/FilterList/FilterList.tsx

```
@@ -7,217 +1,259 @@
+          </Button>
+        </FilterSelection>
+      </div>
+      {filters.map((filter: any, filterIndex: any) =>
+        !filter.isEvent ? (
+          <div
+            className={'py-2 hover:bg-active-blue px-5'}
+            style={{
+              marginLeft: '-1.25rem',
+              width: 'calc(100% + 2.5rem)',
+            }}
+          >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)